build.fish

```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env fish
+
+set AWS_CLI_MFA_SHELL fish; and \
+string replace --all '"' '\\"' < ./src/aws_cli_mfa.py | sed -e "/#INSERT_PYTHON_CODE_HERE/r /dev/stdin" -e "s///" ./src/mfa.fish |
+    sudo tee /usr/local/bin/aws-cli-mfa >/dev/null &&
+    sudo chmod +x /usr/local/bin/aws-cli-mfa &&
+    cp /usr/local/bin/aws-cli-mfa ./bin/aws-cli-mfa-$AWS_CLI_MFA_SHELL
```

Comment:
Yeah, I am pretty sure this is a result of me writing these instructions once, for the original bash, and then when I did it for other shells, I just copy-pasted and did not think about it hard enough. I support cleaning this up as you have suggested in build.sh, and doing it here as well. Please do add the chmod +x as well tho, as noted below

Reply:
Don't you want to change the steps here to match what you did in build.sh?

Reply:
We don't need the additional fish build script anymore, so I updated the README so that the user doesn't need to set a env variable to do the copy (which was what was breaking it for fish).  LMK if you don't like that change.